polaris-service/src/main/java/org/apache/polaris/service/catalog/BasePolarisCatalog.java

```diff
@@ -2055,8 +2054,6 @@ private List<TableIdentifier> listTableLike(PolarisEntitySubType subType, Namesp
    */
   private FileIO loadFileIO(String ioImpl, Map<String, String> properties) {
     Map<String, String> propertiesWithS3CustomizedClientFactory = new HashMap<>(properties);
-    propertiesWithS3CustomizedClientFactory.put(
-        S3FileIOProperties.CLIENT_FACTORY, PolarisS3FileIOClientFactory.class.getName());
```

Comment:
No longer required because of this : https://github.com/apache/iceberg/pull/11259/files

Reply:
Might have to mark this deprecated first since it's a public class

Reply:
I think it was only ever used for this purpose, so we're probably fine